DuckDuckGo/ContentBlocker.storyboard

```diff
@@ -163,31 +163,6 @@
                                             </constraints>
                                         </tableViewCellContentView>
                                     </tableViewCell>
-                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" selectionStyle="none" indentationWidth="10" id="QAm-5P-Wmg">
```

Comment:
Totally agree! I was being cheeky and leaving it in as my current task (site rating) introduces a new row so still needs the current size. Happy to remove it and just reintroduce it again in the next task.